dev-lang/clojure/clojure-1.10.0.ebuild

```diff
@@ -32,17 +29,653 @@ S="${WORKDIR}/clojure-clojure-76b87a6"
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-goodies:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-groovy:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-guice:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-httpclient:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-jetty:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-logging:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-maven:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-metrics:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-osgi:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-other:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-plexus:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-rest:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-shiro:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport-testing:pom:2.9.1-02
+	org.sonatype.nexus.buildsupport:nexus-buildsupport:pom:2.9.1-02
+	org.sonatype.nexus.maven:nexus-common:jar:1.6.7
+	org.sonatype.nexus.maven:nexus-maven-plugins:pom:1.6.7
+	org.sonatype.nexus.maven:nexus-staging:pom:1.6.7
+	org.sonatype.nexus.plugins:nexus-plugins-restlet1x:pom:2.9.1-02
+	org.sonatype.nexus.plugins:nexus-plugins:pom:2.9.1-02
+	org.sonatype.nexus.plugins:nexus-restlet1x-model:jar:2.9.1-02
+	org.sonatype.nexus:nexus-client-core:jar:2.9.1-02
+	org.sonatype.nexus:nexus-components:pom:2.9.1-02
+	org.sonatype.nexus:nexus-oss:pom:2.9.1-02
+	org.sonatype.oss:oss-parent:pom:5
+	org.sonatype.oss:oss-parent:pom:7
+	org.sonatype.plexus:plexus-build-api:jar:0.0.4
+	org.sonatype.plexus:plexus-cipher:jar:1.4
+	org.sonatype.plexus:plexus-cipher:jar:1.7
+	org.sonatype.plexus:plexus-sec-dispatcher:jar:1.3
+	org.sonatype.plexus:plexus-sec-dispatcher:jar:1.4
+	org.sonatype.plugins:nexus-staging-maven-plugin:jar:1.6.7
+	org.sonatype.sisu.inject:guice-bean:pom:1.4.2
+	org.sonatype.sisu.inject:guice-plexus:pom:1.4.2
+	org.sonatype.sisu.siesta:siesta-client:jar:1.7
+	org.sonatype.sisu.siesta:siesta-common:jar:1.7
+	org.sonatype.sisu.siesta:siesta-jackson:jar:1.7
+	org.sonatype.sisu.siesta:siesta:pom:1.7
+	org.sonatype.sisu:sisu-guice:jar:noaop:2.1.7
+	org.sonatype.sisu:sisu-inject-bean:jar:1.4.2
+	org.sonatype.sisu:sisu-inject-plexus:jar:1.4.2
+	org.sonatype.sisu:sisu-inject:pom:1.4.2
+	org.sonatype.sisu:sisu-parent:pom:1.4.2
+	org.sonatype.spice.zapper:spice-zapper:jar:1.3
+	org.sonatype.spice:spice-parent:pom:10
+	org.sonatype.spice:spice-parent:pom:12
+	org.sonatype.spice:spice-parent:pom:15
+	org.sonatype.spice:spice-parent:pom:16
+	org.sonatype.spice:spice-parent:pom:17
+	org.vafer:jdependency:jar:1.2
+	oro:oro:jar:2.0.8
+	xmlpull:xmlpull:jar:1.1.3.1
+	xml-apis:xml-apis:jar:1.0.b2
+	xml-apis:xml-apis:jar:2.0.2
+	xpp3:xpp3_min:jar:1.1.4c
+)
+
+__extract_info() {
+	local f="$1" host="$2" artifact="$3"
```

Comment:
How about replacing the handiwork below with something like:

    local IFS=:
    set -- ${3}
    ${f} "${host}" "${@}" || die

Reply:
Thanks!  I'll need to handle the optional `classifier`, but that's quite possible.